src/exe/main.cc

```diff
@@ -301,6 +302,18 @@ static int start_profiler_internal(DDProfContext *ctx, bool &is_profiler) {
     }
   }
 
+  LG_NTC("Starting profiler");
+
+  // Simulate startup slowdown if requested
+  if (const char *s = getenv(k_startup_wait_ms_env_variable); s != nullptr) {
```

Comment:
minor: this could be extracted to a function
Also I'm curious on why we need to slow startup ? Is this to test the sequence in which we are missing mmap events ?

Reply:
This is to test reading files from profiler mount namespace when process has finished (will come in next PRs;) )

Reply:
oooh, wow, that is a cool feature

Reply:
I extracted code to a function